lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1880,6 +1880,9 @@ sub investigate {
             $inv{needles_log} ||= 'No needle changes recorded, test regression due to needles unlikely';
             $inv{needles_diff_stat} = $self->git_diff($dir, $refspec_needles_range) if $inv{needles_log};
         }
+        my @files_packages = map { Mojo::File->new($_->result_dir(), 'worker_packages.txt')->slurp } ($prev, $self);
+        my $diff_packages  = eval { diff(\$files_packages[0], \$files_packages[1], {CONTEXT => 0}) };
+        $inv{diff_packages_to_last_good} = join("\n", grep { !/(^@@|$ignore)/ } split(/\n/, $diff_packages));
```

Comment:
please add a test for both the cases of the file not existing at all (due to PACKAGES_CMD not being defined) and the case of an existing list in `$prev`, e.g. just package `foo-1.0` and a mocked command giving back a static list, e.g. "foo-1.1\nbar-42" for `$self`

Reply:
Ok, thanks